docs/README.md

```diff
@@ -17,28 +17,20 @@ and diagrams are generated, even undocumented classes / functions. This gives ab
 
 ### mkdocs
 
-To build mkdocs (to `/tmp/docs_build/` as an example):
-
-```
-mkdir /tmp/docs_build/
-```
-
-#### Working in Man Group:
-
-To build:
+Install
 ```
-docker run --rm -v /tmp/docs_build:/tmp/docs_build -v $(pwd)/mkdocs:/docs external-sandbox-docker.repo.prod.m/squidfunk/mkdocs-material:latest build -f mkdocs.yml -d /tmp/docs_build
+pip install mkdocs-material mkdocs-jupyter
 ```
 
-To run the web server on port 8000 (from the `docs` directory):
+To build mkdocs (to `/tmp/docs_build/` as an example):
 ```
-docker run --rm -p 8000:8000 -v ${PWD}/mkdocs:/docs external-sandbox-docker.repo.prod.m/squidfunk/mkdocs-material:latest serve -f mkdocs.yml -a 0.0.0.0:8000
+cd docs/mkdocs
+mkdocs build -d /tmp/docs_build
```

Comment:
Need not be addressed: I get the following warnings when I build.

<details>
<summary>Traceback</summary>

```
WARNING -  A relative path to './api/index.html' is included in the 'nav' configuration, which is not found in the documentation files.
INFO    -  Doc file 'error_messages.md' contains an absolute link '/technical/on_disk_storage/#data-layer', it was left as is. Did you mean 'technical/on_disk_storage.md#data-layer'?
INFO    -  Doc file 'error_messages.md' contains an absolute link '/technical/on_disk_storage/#index-layer', it was left as is. Did you mean 'technical/on_disk_storage.md#index-layer'?
INFO    -  Doc file 'error_messages.md' contains an absolute link '/api/query_builder', it was left as is.
INFO    -  Doc file 'error_messages.md' contains an absolute link '/faq#does-arcticdb-support-categorical-data', it was left as is. Did you mean 'faq.md#does-arcticdb-support-categorical-data'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../history', it was left as is. Did you mean 'history.md'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../', it was left as is. Did you mean 'index.md'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../technical/on_disk_storage', it was left as is. Did you mean 'technical/on_disk_storage.md'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../technical/on_disk_storage', it was left as is. Did you mean 'technical/on_disk_storage.md'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../#reading-and-writing-dataframes', it was left as is. Did you mean 'index.md#reading-and-writing-dataframes'?
INFO    -  Doc file 'faq.md' contains an unrecognized relative link '../technical/on_disk_storage', it was left as is. Did you mean 'technical/on_disk_storage.md'?
INFO    -  Doc file 'faq.md' contains an absolute link '/api/arcticdb/arcticdb.LibraryOptions', it was left as is.
INFO    -  Doc file 'faq.md' contains an absolute link '/runtime_config#versionmapreloadinterval', it was left as is. Did you mean 'runtime_config.md#versionmapreloadinterval'?
INFO    -  Doc file 'faq.md' contains an absolute link '/runtime_config#logging-configuration', it was left as is. Did you mean 'runtime_config.md#logging-configuration'?
INFO    -  Doc file 'faq.md' contains an absolute link '/runtime_config#versionstorenumcputhreads-and-versionstorenumiothreads', it was left as is. Did you mean 'runtime_config.md#versionstorenumcputhreads-and-versionstorenumiothreads'?
INFO    -  Doc file 'faq.md' contains an absolute link '/api/arcticdb/arcticdb.LibraryOptions', it was left as is.
INFO    -  Doc file 'history.md' contains an unrecognized relative link '../licensing', it was left as is. Did you mean 'licensing.md'?
INFO    -  Doc file 'lib_config.md' contains an absolute link '/api/arcticdb/arcticdb.LibraryOptions', it was left as is.
INFO    -  Doc file 'runtime_config.md' contains an absolute link '/technical/on_disk_storage#symbol-list-caching', it was left as is. Did you mean 'technical/on_disk_storage.md#symbol-list-caching'?
INFO    -  Doc file 'runtime_config.md' contains an absolute link '/technical/on_disk_storage#version-layer', it was left as is. Did you mean 'technical/on_disk_storage.md#version-layer'?
INFO    -  Converting notebook (execute=False): /home/jjerphan/dev/ArcticDB/docs/mkdocs/docs/notebooks/ArcticDB_demo_lmdb.ipynb
INFO    -  Doc file 'technical/architecture.md' contains an unrecognized relative link '../on_disk_storage', it was left as is. Did you mean 'on_disk_storage.md'?
INFO    -  Doc file 'technical/on_disk_storage.md' contains an absolute link '/api/library', it was left as is.
INFO    -  Doc file 'technical/on_disk_storage.md' contains an absolute link '/api/arcticdb/arcticdb.LibraryOptions', it was left as is.
INFO    -  Doc file 'technical/on_disk_storage.md' contains an absolute link '/runtime_config', it was left as is. Did you mean '../runtime_config.md'?
INFO    -  Doc file 'tutorials/fundamentals.md' contains an unrecognized relative link '../../#slicing-and-filtering', it was left as is. Did you mean '../index.md#slicing-and-filtering'?
INFO    -  Doc file 'tutorials/parallel_writes.md' contains an unrecognized relative link '../../faq#how-does-arcticdb-handle-concurrent-writers', it was left as is. Did you mean '../faq.md#how-does-arcticdb-handle-concurrent-writers'?
```

</details>

Reply:
Thanks, will address as a follow up